packages/@coorpacademy-components/src/molecule/learner-skill-card/index.js

```diff
@@ -0,0 +1,169 @@
+import React, {useCallback, useState} from 'react';
+import {convert} from 'css-color-function';
+import {get, getOr} from 'lodash/fp';
+import PropTypes from 'prop-types';
+import Icon from '../../atom/icon';
+import ButtonLink from '../../atom/button-link';
+import Provider from '../../atom/provider';
+import style from './style.css';
+
+const LearnerSkillCard = (props, context) => {
+  const {
+    'aria-label': ariaLabel,
+    skillCourses,
+    skillQuestions,
+    skillTitle,
+    skillAriaLabel,
+    completedCourses,
+    buttonReviewLabel,
+    buttonReviewAriaLabel,
+    buttonExploreLabel,
+    buttonExploreAriaLabel,
+    onReviewClick,
+    onExploreClick
+  } = props;
+  const {skin, translate} = context;
+  const [hovered, setHovered] = useState(false);
+  const primarySkinColor = getOr('#0061FF', 'common.primary', skin);
+
+  const courses = translate('courses');
+  const questions = translate('questions');
+  const skillFocus = translate('skill_focus');
+  const coursesCompleted = translate('courses_completed');
+
+  const handleMouseOver = useCallback(() => setHovered(true), [setHovered]);
+
+  const handleMouseLeave = useCallback(() => setHovered(false), [setHovered]);
+
+  const buttonReviewProps = {
+    customStyle: {
+      backgroundColor: '#FFF',
+      transition: 'background-color 0.15s ease-in-out, color 0.15s ease-in-out'
+    },
+    onReviewClick,
+    'aria-label': buttonReviewAriaLabel ? `${skillTitle}, ${buttonReviewAriaLabel}` : skillTitle,
+    label: buttonReviewLabel,
+    'data-name': 'learner-skill-card-review-button'
+  };
+
+  const buttonExploreProps = {
+    customStyle: {
+      backgroundColor: hovered ? primarySkinColor : convert(`color(${primarySkinColor} a(0.07))`),
+      color: hovered ? '#FFFFFF' : primarySkinColor,
+      transition: 'background-color 0.15s ease-in-out, color 0.15s ease-in-out'
+    },
+    onExploreClick,
+    'aria-label': buttonExploreAriaLabel ? `${skillTitle}, ${buttonExploreAriaLabel}` : skillTitle,
+    label: buttonExploreLabel,
+    'data-name': 'learner-skill-card-explore-button',
+    icon: {
+      position: 'left',
+      type: 'compass',
+      customStyle: {
+        height: 16,
+        width: 16
+      }
+    }
+  };
+
+  const completedPercentage = Number.parseInt((completedCourses / skillCourses) * 100);
+
+  const ProgressBar = useCallback(() => {
```

Comment:
there is already a ProgressBar component, I don't know if you checked it:
src/molecule/progress-bar/index.js

Reply:
I did check it, but the UI is a bit different, so I'm not sure if we really want to modify and make the original component more complex just for for this use case.